packages/climate-ref-pmp/src/climate_ref_pmp/diagnostics/enso.py

```diff
@@ -0,0 +1,171 @@
+from EnsoMetrics.EnsoComputeMetricsLib import ComputeCollection  # How can I import this?
+from pcmdi_metrics.enso.lib import metrics_to_json
+from pcmdi_metrics import resources
```

Comment:
Since these have conda-only dependencies, you will need to write a driver script (it can live in this repository if you want). This can then be run inside the pmp conda environment (once you have added the enso package to the requirements for that).

The driver will likely read a JSON file and then run lines 141 to 170

We can't run everything in the main process because we can't make a single conda env with all providers dependencies in (nor do I want to maintain that).

Reply:
@lewisjared thank you for the advice. I will write a driver that lives in REF repo for now but can be migrated to PMP later.

Reply:
@lewisjared in commit https://github.com/Climate-REF/climate-ref/pull/273/commits/b2aeb1c0866e07c57222f57a58ba7476f54931fa I added a driver file living in this repo. Could you help building a commend line to run this driver using the pmp conda env?

Reply:
I've pushed a commit which points you in the right direction. I don't have the right obs data locally to actually test this though